docs/firststeps/nextsteps.md

```diff
@@ -24,19 +24,17 @@ hide:
 
 <!-- Great! Now you are ready to start learning about the LUMI environment and how to work in it. On this page are given short descriptions of each menu topic of this documenation. -->
 
-This is a short introduction to the other menu topics in this documentation:
+In order to continue your endeavour into the use of LUMI, you may take a look at:
 
+- The [**Hardware**][Hardware] section to get to know the LUMI system architecture and the different hardware partitions available on LUMI.
 
-- To get to know the LUMI system architecture and different partitions available on LUMI, continue to the [**Hardware**][Hardware] section.
+- The [**Run jobs**][running-jobs] section to identify ways to run computations on LUMI.
```

Comment:
```suggestion
- The [**Run jobs**][running-jobs] section to identify ways to run computations on LUMI. Here you find also the information about which locations on LUMI are suitable for storing data, and how the billing policy goes. 
```

Reply:
I have added the information about data storage options and billing policy.